### ISSUE TITLE
lags on chrome

### ISSUE THREAD

**Issue by ak9250** [FullScreenShenanigans/FullScreenMario#356]
there is a lag in gameplay on chrome

**Comment by JoshuaKGoldberg** [comment]
Yup :frowning: there is lag, and the engine needs a lot of performance work.

To mitigate it, you could try:
* Going to Options > Framerate and changing it to 30fps
* Reducing the width of your screen

If lag is particularly bad during anything, you can post it here and I'll take a look.

**Comment by mutdmour** [comment]
lag is noticeable when nearing Bowser on the first two levels that I tried
